_includes/header.html

```diff
@@ -11,7 +11,11 @@
       <a class="nav-link" href="sign_up_form.html">Sign up form</a>
     </li>
     <li class="nav-item">
+<<<<<<< HEAD
```

Comment:
This commits seems to only contain, merge conflicts, these need resolving before they are committed.